documentation/src/docs/asciidoc/release-notes/release-notes-5.8.0-M1.adoc

```diff
@@ -90,6 +90,9 @@ on GitHub.
 * `DynamicTests.stream()` can now consume `Named` input and will use each name-value pair
   as the display name and value for each generated dynamic test (see
   <<../user-guide/index.adoc#writing-tests-dynamic-tests-examples,User Guide>> for details).
+* New `assertThrowsExactly` method is introduced as a more strict version of `assertThrows`
```

Comment:
This also needs to be moved to `release-notes-5.8.0-M2.adoc`.

Reply:
Done.